scrapy/__init__.py

```diff
@@ -14,8 +14,8 @@
 
 # Check minimum required Python version
 import sys
-if sys.version_info < (3, 5):
-    print("Scrapy %s requires Python 3.5" % __version__)
+if sys.version_info < (3, 5, 1):
```

Comment:
Hm, what do you think about allowing it to run in 3.5.0? Most likely, it'd work, so this change may break Scrapy for some users.